service/src/main/java/bio/terra/externalcreds/controllers/OidcApiController.java

```diff
@@ -104,8 +104,9 @@ private LinkInfo getLinkInfoFromLinkedAccount(LinkedAccount linkedAccount) {
         providerService.createLink(
             provider, userId, oauthcode, redirectUri, Set.copyOf(scopes), state);
 
-    return ResponseEntity.ok(
-        getLinkInfoFromLinkedAccount(linkedAccountWithPassportAndVisas.getLinkedAccount()));
+    return ResponseEntity.of(
+        linkedAccountWithPassportAndVisas.map(
+            x -> getLinkInfoFromLinkedAccount(x.getLinkedAccount())));
```

Comment:
this could also be written as
```
    return ResponseEntity.of(
        linkedAccountWithPassportAndVisas
            .map(LinkedAccountWithPassportAndVisas::getLinkedAccount)
            .map(this::getLinkInfoFromLinkedAccount));
```
Which do you like better?

Reply:
Either works for me!

Reply:
I'm more familiar with the syntax of what you have now, but either would be fine with me!